types/progressbar.js/progressbar.js-tests.ts

```diff
@@ -0,0 +1,66 @@
+import ProgressBar = require('progressbar.js');
+import utils = require('progressbar.js/utils');
```

Comment:
this is incorrect import..
```js
const utils = require('progressbar.js/utils');
```

results in 
```
c:\temp\test2>node a.js
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module 'progressbar.js/utils'
Require stack:
- c:\temp\test2\a.js
←[90m    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:686:27)←[39m
←[90m    at Module.require (internal/modules/cjs/loader.js:848:19)←[39m
←[90m    at require (internal/modules/cjs/helpers.js:74:18)←[39m
    at Object.<anonymous> (c:\temp\test2\a.js:2:15)
←[90m    at Module._compile (internal/modules/cjs/loader.js:955:30)←[39m
←[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)←[39m
←[90m    at Module.load (internal/modules/cjs/loader.js:811:32)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:723:14)←[39m
←[90m    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)←[39m {
  code: ←[32m'MODULE_NOT_FOUND'←[39m,
  requireStack: [ ←[32m'c:\\temp\\test2\\a.js'←[39m ]
}
```

Reply:
let me check, thanks!

Reply:
@sheetalkamat I amended with proper change (now I've just do the homework adding this into real progressbar.js code). Thanks for review!